### PR TITLE
Implemented ignored errors functionality

### PR DESCRIPTION
**Pull request by anthonyraymond**

### CODE REVIEW

File: lib/init.coffee [hunk 1]
```diff
@@ -10,6 +10,8 @@ module.exports =
       title: 'gjslint executable path'
       default: 'gjslint'
     gjslintIgnoreList:
+      title: 'Errors to ignore'
```
Comment (Arcanemagus) [inline lib/init.coffee]: This needs much better wording, you shouldn't rely on users reading the README as most will not.

File: README.md [hunk 2]
```diff
@@ -11,14 +11,17 @@ Before using this plugin, you must ensure that `gjslint` is installed on your sy
...
-  'executablePath': '' #gjslint path. run 'which gjslint' to find the path
-  'gjslintIgnoreList': [] # Currently not implemented: gjslint ignore codes from http://goo.gl/OhYHYl
-  'flags': [
-    '--flagfile=.gjslintrc',
-    '--strict'
+"linter-gjslint":
+   executablePath: "" #gjslint path. run 'which gjslint' to find the path
+   gjslintIgnoreList: [
+    "110"
+  ] # gjslint ignore codes from http://goo.gl/OhYHYl
```
Comment (Arcanemagus) [inline README.md]: You should be linking directly to https://github.com/google/closure-linter/blob/v2.3.19/closure_linter/errors.py (note the tagged release and not master, that way if master changes your link doesn't break).

File: README.md [hunk 3]
```diff
@@ -11,14 +11,17 @@ Before using this plugin, you must ensure that `gjslint` is installed on your sy
 Run `amp install linter-gjslint` or search for `linter-gjslint` in Atom package manager.
 
 ## Settings
-You can configure linter-gjslint by editing ~/.atom/config.cson (choose Open Your Config in Atom menu):
+You can configure linter-gjslint by accessing the options in File > Settings > Packages.<br>
+Or by editing ~/.atom/config.cson (choose Open Your Config in Atom menu):
 ```
```
Comment (Arcanemagus) [inline README.md]: Specify `CoffeeScript` (or one of the aliases [here](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L569)) to enable syntax highlighting.

File: README.md [hunk 4]
```diff
@@ -11,14 +11,17 @@ Before using this plugin, you must ensure that `gjslint` is installed on your sy
...
-'linter-gjslint':
-  'executablePath': '' #gjslint path. run 'which gjslint' to find the path
-  'gjslintIgnoreList': [] # Currently not implemented: gjslint ignore codes from http://goo.gl/OhYHYl
-  'flags': [
-    '--flagfile=.gjslintrc',
-    '--strict'
+"linter-gjslint":
+   executablePath: "" #gjslint path. run 'which gjslint' to find the path
```
Comment (Arcanemagus) [inline README.md]: This should show the default value of `gjslint`.

File: README.md (hunk 3 above)
Comment (anthonyraymond) [inline README.md]: Alright, i'll do this tomorrow, i'm on my phone now.   Thanks for your time.

**Comment by anthonyraymond** [comment]
I believe i have changed all you asked for.

File: README.md [hunk 5]
```diff
@@ -11,14 +11,15 @@ Before using this plugin, you must ensure that `gjslint` is installed on your sy
...
-  'executablePath': '' #gjslint path. run 'which gjslint' to find the path
-  'gjslintIgnoreList': [] # Currently not implemented: gjslint ignore codes from http://goo.gl/OhYHYl
-  'flags': [
-    '--flagfile=.gjslintrc',
-    '--strict'
+You can configure linter-gjslint by accessing the options in File > Settings > Packages.<br>
+Or by editing ~/.atom/config.cson (choose Open Your Config in Atom menu):
+```CoffeeScript
+"linter-gjslint":
+   executablePath: "" #gjslint path. run 'which gjslint' to find the path
```
Comment (Arcanemagus) [inline README.md]: Extra space at the start of this line and the next.

**Comment by Arcanemagus** [comment]
LGTM after that last README issue gets fixed :wink:.

**Comment by anthonyraymond** [comment]
Extra space removed :smile:

**Comment by Arcanemagus** [comment]
Thanks!

**Comment by Arcanemagus** [comment]
@artoale there's new functionality here, shall I push out a new release or do you have something else in mind before that?

**Comment by artoale** [comment]
@landon if it's working it's looking good for a release to me! :shipit:

**Comment by Arcanemagus** [comment]
v1.4.0 published including this :wink:.

**Comment by anthonyraymond** [comment]
Thanks :wink:

**Pull request merged**